matchstart.fbs

```diff
@@ -6,9 +6,16 @@ table RLBot {}
 /// A normal human player
 table Human {}
 
+enum PsyonixSkill : ubyte {
+  Beginner,
```

Comment:
I didn't know we could spawn beginner bots

Reply:
yep!